src/java.desktop/share/classes/sun/awt/util/IdentityLinkedList.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2007, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2007, 2021 Oracle and/or its affiliates. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 2007, 2021, Oracle and/or its affiliates. All rights reserved.
```
There should be another comma.

Reply:
Other than that, the changes look good.